src/server/NetworkingServer.cpp

```diff
@@ -163,36 +180,53 @@ void NetworkingServer::ComputeGameState() {
         mAcceptingMoveCommands = true;
         std::this_thread::sleep_for(std::chrono::milliseconds(100));
         mAcceptingMoveCommands = false;
-        // update game state, take moves array and simulate next tick given current state and inputs
-        // ... simplest behavior would be to track client locations and apply vectors to them ...
 
-        if (mClientMoves.size() == 0) {
-            continue;
-        }
-        std::cout << "game state thread: dumping client moves for tick " << mCurrTick << std::endl;
-        for (const auto iter : mClientMoves) {
-            sf::Uint32 client_id = iter.first;
-            float* move_vector = iter.second;
-            if (mClientPositions.find(client_id) != mClientPositions.end()) {
-                mClientPositions[client_id][0] += move_vector[0]*5.f; // x component
-                mClientPositions[client_id][1] += move_vector[1]*5.f; // y component
-            }
-            std::cout << iter.first << " " << iter.second[0] << " " << iter.second[1] << std::endl;
-        }
-        std::cout << "game state thread: dumping client positions for tick " << mCurrTick << std::endl;
-        for (const auto iter : mClientPositions) {
-            std::cout << iter.first << " " << iter.second[0] << " " << iter.second[1] << std::endl;
-        }
         // window is closed, time to reset. May want to use fix sized array as an optimization.
         // It's a bit mean to disguard old commands, maybe there should be windows for multiple ticks?
```

Comment:
wait all I see here is deleted code...  don't you need something to call something here? if this is simply the "game clock" then it should be renamed from computeGameState, to runGameClock or something

Reply:
Move the tick update to `GameController` in b5490ec